lib/ood_core/job/adapters/kubernetes/templates/pod.yml.erb

```diff
@@ -10,6 +10,10 @@ metadata:
     <%- if !script.accounting_id.nil? && script.accounting_id != "" -%>
     account: <%= script.accounting_id %>
     <%- end -%>
+  annotations:
+    <%- unless script.wall_time.nil? -%>
+    pod.kubernetes.io/lifetime: <%= seconds_to_duration(script.wall_time) %>
```

Comment:
This needs to be `helper.seconds_to_duration(script.wall_time)` otherwise we get undefined method error (because the templating context is the batch class).

Reply:
Made that change